demo/url-bar.html

```diff
@@ -20,11 +20,13 @@
   <link rel="import" href="../url-bar.html">
   <link rel="import" href="../demo-pages-shared-styles.html">
 
-  <style is="custom-style" include="demo-pages-shared-styles">
-    iframe {
-      width: 100%;
-    }
-  </style>
+  <custom-style include="demo-pages-shared-styles">
+    <style is="custom-style" include="demo-pages-shared-styles">
```

Comment:
having `include` in both `<style>` and `<custom-style>` is kinda confusing, could we keep it only in the `<style>` element?